qcodes/instrument_drivers/ZI/ZIUHFLI.py

```diff
@@ -1631,6 +1632,25 @@ def __init__(self, name: str, device_ID: str, **kwargs) -> None:
                            parameter_class=Scope,
                            )
 
+        ########################################
+        # SYSTEM PARAMETERS
+        self.add_parameter('external_clock_source',
+                           set_cmd=partial(self.daq.setInt,
```

Comment:
shouldn't a value of some other clock-related parameter(s) be updated and/or invalidated once this gets set to True?

Reply:
No, it has no other effect on the device or device parameters.